src/backend/libc/io/syscalls.rs

```diff
@@ -4,6 +4,8 @@ use crate::backend::c;
 #[cfg(not(target_os = "wasi"))]
 use crate::backend::conv::ret_discarded_fd;
 use crate::backend::conv::{borrowed_fd, ret, ret_c_int, ret_owned_fd, ret_usize};
+#[cfg(any(target_os = "android", target_os = "linux"))]
```

Comment:
Here and elsewhere, rustix has a macro providing a shorthand for `any(target_os = "android", target_os = "linux")`, which is `linux_kernel`.